statute.md

```diff
@@ -0,0 +1,106 @@
+## Tech Workers Coalition Berlin
+# Statute
+
+# 1 
+
+1.1
+
+This document defines:
+* Decision making process of Berlin Tech Workers Coalition (Berlin TWC from now on)
+* How to become a member, and lose membership
+* Working Teams: scope, creation process, disband process
+* Credentials management and sharing
+
+1.2
+
+The Statute exists to foster participation in Berlin TWC by creating a transparent and democratic decisional and organizational framework. 
+ 
+1.3
+Berlin TWC has as its primary goal the empowerment of Tech Workers in their industry and in the society at large, in solidarity with other workers. Power here is understood as the capability of collectives to shape their own companies, governments. This goal will be pursued through workers’ organization, workers’ education and the progressive liberation of technological production.
```

Comment:
`Berlin TWC has as its primary goal, the empowerment of Tech Workers in their industry and in society at large, in solidarity with all workers.`

It's important to me, that we don't `other` other workers, especially when industry is not clearly defined either, nor is tech worker defined.

Reply:
For that I wrote a very explicit sentence in the PR that is already open.